redis/src/cluster_async/mod.rs

```diff
@@ -819,8 +823,25 @@ impl<C> Future for Request<C> {
                 }
                 .into();
             }
+            RequestStateProj::UpdateMoved { future } => {
+                if let Err(err) = ready!(future.poll(cx)) {
+                    // Updating the slot map based on the MOVED error is an optimization.
+                    // If it fails, proceed by retrying the request with the redirected node,
+                    // and allow the slot refresh task to correct the slot map.
+                    warn!(
+                        "Failed to update the slot map based on the received MOVED error.\n
```

Comment:
1. Avoid using `\n` in the log as it create line without its prefix. Please use 2 x `warn!` calls
2. If it is OK to fail, why use `warn!`? (this is a good place to use telemetry call - when its available)

Reply:
ack, changed to info!